main.py

```diff
@@ -3,6 +3,14 @@
 data = []
 with open('real-estate-sample.csv', 'r') as file:
```

Comment:
this is repetitive you can create a function that takes in file name and mode and updates accordingly